lstchain/mc/mc.py

```diff
@@ -66,7 +67,7 @@ def int_diff_sp(emin, emax, sp_idx, e0):
 
     return integral_e
 
-def rate(emin, emax, param, cone, area):
+def rate(shape, emin, emax, param, cone, area):
```

Comment:
The param argument does not seem to be explained. Especially important to avoid confusion with the sign conventions of the spectral index and the curvature index!  I think in each function dealing with spectral models it should be explained explicitly in the comments what is the meaning of the sign of the parameters.